### ISSUE TITLE
Can't find $ref

### ISSUE THREAD

**Issue by C4st3ll4n** [Redocly/redocly-cli#984]
After the update to the version [1.0.0-beta.119](https://www.npmjs.com/package/@redocly/cli/v/1.0.0-beta.119), the 'lint' throws some fails on valid swagger

output on beta-119
```
  ]
}< ... 322 more problems hidden > increase with `--max-problems N`
Mova_Swagger_API_Pagamentos.json: validated in 306ms

❌ Validation failed with 6 errors and 416 warnings.
run `openapi lint --generate-ignore-file` to add all problems to the ignore file.


```

output on 1.0.0-beta.118
```
 ]
}< ... 322 more problems hidden > increase with `--max-problems N`
.\Mova_Swagger_API_Pagamentos.json: validated in 315ms

Woohoo! Your OpenAPI definition is valid. 🎉
You have 422 warnings.


```

**Comment by adamaltman** [comment]
We are working on the patch. 

cc @AntonKozachuk

**Comment by ZyzychO** [comment]
Resolved for the client in Redocly CLI version 1.1.0-beta.120
